### PR TITLE
Merge comments

### PR DESCRIPTION
**Pull request by garber34**
Made edits based on Stanley and Eliot's code review comments

### CODE REVIEW

File: client/components/Dropdown.js [hunk 1]
```diff
@@ -1,66 +1,64 @@
...
-        } else {
-            this.setState({
-                show: 'show'
-            })
-        }
-    }
+  constructor() {
+    super();
+    this.state = {
+      show: zz,
```
Comment (OneExceptionn) [inline client/components/Dropdown.js]: just a little confused on what 'zz' is on this line

File: client/components/Profile.js [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
-      )
-
-       
-    };
+  constructor() {
+    super();
+  }
+
+  componentDidMount() {
+    // this.props.getProduct(this.props.match.params.productId)
```
Comment (OneExceptionn) [inline client/components/Profile.js:10]: is the getProduct here just used to test if the profile component is working? I see this in the in the Register component as well.

Comment (garber34) [reply client/components/Profile.js:10]: i don't know. these are in a lot of the components, but i'm not touching them for this branch

File: client/components/Dropdown.js (hunk 1 above)
Comment (garber34) [inline client/components/Dropdown.js]:
```suggestion
      show: ""
```

Comment (garber34) [reply client/components/Dropdown.js]: it's an error. i just reverted it so should be good to go now.

**Review by OneExceptionn — APPROVED** [review]

**Pull request merged**